plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -52,7 +54,10 @@ class DistributionConfig:
 class OptunaConfig:
 
     # Direction of optimization
+    # For single objective optimization
     direction: Direction = Direction.minimize
+    # For multi-objective optimization. If exists, direction will be ignored.
+    directions: Optional[List[Direction]] = None
```

Comment:
Another approach is to make direction effectively:
```python
direction: Union[Direction, List[Direction] = Direction.minimize
```

Since OmegaConf currently does not support Union (it's planned for the next version), you can use Any for now and perform the type checks yourself.
```python
# Union[Direction, List[Direction]
direction: Any = Direction.minimize
```

Once OmegaConf adds support for Union, we can follow up and take advantage of it here.

Reply:
That makes sense.
I removed the `directions` according to your suggestion.
BTW, I sometimes got `direction` value as `str` instead of `Direction`, and I need to add type checking as follows.

```python
            if isinstance(self.optuna_config.direction, str):
                directions = [self.optuna_config.direction]
            else:
                directions = [self.optuna_config.direction.name]
```
(https://github.com/toshihikoyanase/hydra/blob/f1eb00018e05f0aaace8a51a85ffddcff4d7959e/plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py#L181-L184)

Do you think it Is an expected behavior?